gatsby-config.js

```diff
@@ -159,5 +159,17 @@ module.exports = {
     },
     'gatsby-plugin-react-helmet',
     'gatsby-plugin-catch-links',
+    {
+      resolve: `gatsby-plugin-manifest`,
+      options: {
+        name: 'React Docs',
+        short_name: 'React',
+        start_url: '/',
+        background_color: '#20232a',
+        theme_color: '#20232a',
+        display: 'standalone',
+        icon: 'src/images/react_logo.png',
```

Comment:
I don't think this is right. The URL needs to be a web accessible address. This is the source location, but you can't load e.g. https://deploy-preview-2377--reactjs.netlify.com/src/images/react_logo.png

So this just makes the "Add to Homescreen" popup spin for a while before it gives up on trying to load the icon.

Reply:
I am using plugin gatsby-plugin-manifest so the plugin is handling that path.

![image](https://user-images.githubusercontent.com/30949385/65904244-b5dccc00-e3db-11e9-9524-a7ac6bd4e730.png)

 So it is simply taking that image and generating required variant sizes before it builds. As you can see in the screenshot it generated those images.

Reply:
Hm, I guess I'm just saying that it doesn't seem to work.

iOS+Safari ignores it entirely and shows a thumbnail of the homepage, and Android+Chrome shows the correct icon for reactjs.org (without this PR even being added).

What am I overlook?

Reply:
I've added some commits can you check now? 
So this repository already had `gatsby-plugin-manifest` installed but it was not used so instead of installing another version I used the same plugin. 
Anyway now I've added a `legacy: true` which is supposed to fix issue for IOS. If that doesn't work then installing latest version of `gatsby-plugin-manifest` would be a better choice.

Reply:
Seems like the behavior is the same as I described before.

Reply:
So the problem was currently there is `gatsby-plugin-manifest@2.0.5` installed which seems to not add apple meta tags in html and since the newer versions of the plugin are dependent on newer versions of other packages. we would have to update a lot of plugins (which may break stuff)

So instead of that I managed to fix things by adding meta tags manually which should work on both ios and android.

Commit: 
7fe5098a6113dfe4546e2ab0a917d81b1af91195

Also, I moved `src/images/react_logo.png` to `static/logo-512x512.png` and added `static/logo-180x180.png` for apple.